sycl/test-e2e/ClusterLaunch/cluster_launch_parallel_for.cpp

```diff
@@ -0,0 +1,90 @@
+// Tests whether or not cluster launch was successful, with the correct ranges
+// that were passed via parallel for overload
+// REQUIRES: cuda
```

Comment:
These new tests should require the extension aspect `REQUIRES: aspect-ext_oneapi_cuda_cluster_group`
To avoid setting this on every test you could alternatively create a `lit.local.cfg` file in this directory with 
```
config.required_features += ['aspect-ext_oneapi_cuda_cluster_group', 'cuda']
```